setup.py

```diff
@@ -29,15 +29,14 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'click==7.0',
+        'click',
         'docker',
-        'pip',
-        'PyYAML',
+        'pyyaml',
         'retrying',
         'requests',
         'scrapinghub>=2.3.1',
         'six>=1.7.0',
-        'tqdm==4.55.1',
+        'tqdm',
```

Comment:
We should probably keep some limit, assuming we do not support just any version.